WebDriverAgentLib/Categories/XCUIApplication+FBHelpers.h

```diff
@@ -27,9 +27,10 @@ NS_ASSUME_NONNULL_BEGIN
 - (BOOL)fb_deactivateWithDuration:(NSTimeInterval)duration error:(NSError **)error;
 
 /**
- Return application elements tree in form of nested dictionaries
+  @param excludedAttributes Set of possible attributes to be excluded i.e frame, enabled, visible, accessible, focused
```

Comment:
Please add: `If set to nil or an empty array then no attributes will be excluded from the resulting JSON`

Reply:
done